src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -595,3 +616,48 @@ def get_files_path(data_path):
                 break
 
     return files_path
+
+
+def _make_symlink(path, source_path, name, index):
+    base_name = os.path.join(source_path, name + '_' + str(index))
+    logging.info('creating symlink between Path {} and destination {}'.format(source_path, base_name))
+    os.symlink(path, base_name)
+
+
+def check_data_redundancy(train_path, validate_path):
```

Comment:
Is the return value used for anything aside from the unit tests? If not, let's check instead that logging.warning is called in place of the true check.

Reply:
@haixiw can you please take a look? Thanks